layouts/shortcodes/observability_pipelines/source_settings/sumo_logic.md

```diff
@@ -0,0 +1 @@
+Optionally, in the **Decoding** dropdown menu, select whether your input format is raw **Bytes**, **JSON**, Graylog Extended Log Format (**Gelf**), or **Syslog**. If no decoding is selected, the decoding defaults to JSON.
```

Comment:
**Gelf** is quoted as UI copy that I don't think I can get to myself to check, so I'm leaving it, but it's usually **GELF**, which is how Graylog refers to it.

Reply:
Ah yeah, agree it should **GELF**...but let's go with what's in the UI for now.